app/assets/javascripts/analytics/static-tracker.js

```diff
@@ -81,6 +82,33 @@
 
   };
 
+  StaticTracker.prototype.callOnNextPage = function(method, params) {
+    params = params || [];
+
+    if (!$.isArray(params)) {
+      params = [params];
+    }
+
+    if (GOVUK.cookie && typeof this[method] === "function") {
+      params.unshift(method);
+      GOVUK.cookie('analytics_next_page_call', params.join('|'));
+    }
+  };
+
+  StaticTracker.prototype.callMethodRequestedByPreviousPage = function() {
+    if (GOVUK.cookie && GOVUK.cookie('analytics_next_page_call') !== null) {
+      var params = GOVUK.cookie('analytics_next_page_call').split('|'),
```

Comment:
I don't know how long these strings are likely to be and how much we can put into a cookie but could we use `JSON.stringify` and `JSON.parse` to manage the encoding and decoding of the data?